controllers/topics/inviteMembers.js

```diff
@@ -84,8 +82,7 @@ module.exports = async (req, res) => {
           }
         );
         await channel.create();
-        await channel.sendMessage({user_id: req?.userId, text: invitations_msg}, {skip_push: true});
-        //end topic invitation message primary chat
+        await channel.sendMessage({user_id: user_id, text: invitations_msg}, {skip_push: true});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `sendMessage` method's `user_id` parameter from `req?.userId` to `user_id` could potentially introduce bugs if `user_id` is not defined or not the intended recipient of the message. Please ensure that `user_id` is correctly defined and is the intended recipient.
```diff
- await channel.sendMessage({user_id: user_id, text: invitations_msg}, {skip_push: true});
+ await channel.sendMessage({user_id: req?.userId, text: invitations_msg}, {skip_push: true});
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->